pwa/src/templates/mappingDetailTemplate/MappingDetailsTemplate.tsx

```diff
@@ -39,15 +37,19 @@ export const MappingDetailTemplate: React.FC<MappingDetailsTemplateProps> = ({ m
   const { toggleDashboardCard, getDashboardCard, loading: dashboardLoading } = useDashboardCard();
   const [testMappingData, setTestMappingData] = React.useState<object>();
 
+  const [code, setCode] = React.useState<string>("");
+
   const queryClient = useQueryClient();
   const getMapping = useMapping(queryClient).getOne(mappingId);
   const deleteMapping = useMapping(queryClient).remove();
   const testMapping = useMapping(queryClient).testMapping(mappingId);
 
   const dashboardCard = getDashboardCard(mappingId);
 
+  const beautify = require("ace-builds/src-noconflict/ext-beautify");
```

Comment:
If this implementation was lost, I don't think someone would understand that _this_ is the expected `beautify` prop, passed to `CodeEditor`. 

See the other comment about extracting all of this logic into another component. Could we simply add this there?

(Do we actually need to pass this?)